types/react-reconciler/index.d.ts

```diff
@@ -275,11 +275,6 @@ declare namespace ReactReconciler {
          */
         noTimeout: NoTimeout;
 
-        /**
-         * You can proxy this to `queueMicrotask` or its equivalent in your environment.
-         */
-        queueMicrotask(fn: () => void): void;
```

Comment:
Wasn't this just renamed to `scheduleMicrotask`? It would be great if you could add `supportsMicrotask` as well (and `getCurrentEventPriority` if you're up to it).

Reply:
scheduleMicrotask,

Reply:
scheduleMicrotask, supportsMicrotask and getCurrentEventPriority is only used in react-reconciler@next (React 18), not current 0.26.2, which unstable release, so i remove these methods.

Reply:
Gotcha, good catch. It's hard to track down what's released and what's not.